content/blog/getting-started-with-hpe-greenlake-api-for-virtualization.md

```diff
@@ -0,0 +1,412 @@
+    "error": null,
+    "estimatedRunningDurationMinutes": 0,
+    "generation": 4,
+    "groups": [
+        {
+            "id": "eb988b5e2dcb11ec840712b3b5263ef4",
+            "name": "Default Group"
+        }
+    ],
+    "healthStatus": "OK",
+    "id": "de6d9725-7803-4ef5-9cce-f40ad05f879f",
+    "logMessages": [
+        {
+            "message": "Terminate: d4c46c6e-51ff-59c0-a24a-ef52d77f0b16 task is created",
+            "timestampAt": "2024-03-27T22:37:20.509674904Z"
+        },
+        {
+            "message": "Terminate: d4c46c6e-51ff-59c0-a24a-ef52d77f0b16 task is running",
+            "timestampAt": "2024-03-27T22:37:20.509678268Z"
+        },
+        {
+            "message": "Terminate: RonD-deploy-CSP-1 task is running",
+            "timestampAt": "2024-03-27T22:37:20.79512845Z"
+        },
+        {
+            "message": "Terminate: RonD-deploy-CSP-1 task is succeeded",
+            "timestampAt": "2024-03-27T22:37:37.652529902Z"
+        }
+    ],
+    "name": "Terminate: RonD-deploy-CSP-1",
+    "parentTask": null,
+    "progressPercent": 100,
+    "recommendations": [],
+    "resourceUri": "/data-services/v1beta1/async-operations/de6d9725-xxxx-xxxx-xxxx-f40ad05f879f",
+    "rootTask": {
+        "id": "de6d9725-7803-4ef5-9cce-f40ad05f879f",
+        "name": "",
+        "resourceUri": "/data-services/v1beta1/async-operations/de6d9725-xxxx-xxxx-xxxx-f40ad05f879f",
+        "type": "task"
+    },
+    "services": [
+        "private-cloud-business-edition"
+    ],
+    "sourceResource": {
+        "name": "RonD-deploy-CSP-1",
+        "resourceUri": "/api/v1/csp-machine-instances/d4c46c6e-xxxx-xxxx-xxxx-ef52d77f0b16",
+        "type": "AWS Instance"
+    },
+    "startedAt": "2024-03-27T22:37:20.509668598Z",
+    "state": "SUCCEEDED",
+    "subtreeTaskCount": 1,
+    "suggestedPollingIntervalSeconds": 30,
+    "type": "task",
+    "updatedAt": "2024-03-27T22:37:37.688700579Z",
+    "userId": "ronald.dharma@hpe.com"
+}
+```
+
+### Wow… that was so cool. What about provisioning  a VMware Datastore on-premises?
+
+Here is another example of using the HPE GreenLake API for virtualization to provision a datastore out of an HPE disaggregated Hyperconverged Infrastructure (dHCI) system for on-premise deployment. The HPE dHCI instance had already been onboarded into the HPE GreenLake for Private Cloud Business Edition service (PCBE).  The [API](https://developer.greenlake.hpe.com/docs/greenlake/services/virtualization/public/openapi/virtualization-public-v1beta1/operation/CreateDS/) that is used for the datastore provisioning is `POST {baseUrl} /virtualization/v1beta1/datastores`, which also required a definition of this JSON structure with values in the **body (Payload)**. The information about this request JSON body is presented in the figure below. 
+
+![](/img/create-datastore-on-dhci.png)
+
+*T﻿he above figure display the required body (Payload) to deploy datastore into a hyper-converged instance with VMware.*
```

Comment:
Change "display" to "displays".  Add the word "used" prior to "to deploy". Add the word "the" prior to "datastore".
i.e. "The above figure displays the required body (Payload) used to deploy the datastore into..."